Scoop/Controller/Onboarding/PreferencesViewController.swift

```diff
@@ -23,9 +23,12 @@ class PreferencesViewController: OnboardingViewController {
                 return
             }
             
-            NetworkManager.shared.currentUser.talkingPref = self.talkativeSlider.value
-            NetworkManager.shared.currentUser.musicPref = self.musicSlider.value
-            
+            //TODO: Not working: Backend still debugging
+            self.addPrompt(name: "Talkative", placeholder: "0", answer: String(self.talkativeSlider.value))
```

Comment:
Do you need `self` here

Reply:
If I remove, it end up getting an error that tells me to add self. I think its because, its called within a UIAction so it needs the explicit mention